kong/utils.go

```diff
@@ -741,3 +741,119 @@ func FillPluginsDefaults(plugin *Plugin, schema Schema) error {
 func FillPluginsDefaultsWithOpts(plugin *Plugin, schema map[string]interface{}, opts FillRecordOptions) error {
 	return fillConfigRecordDefaultsAutoFields(plugin, schema, opts)
 }
+
+func deleteAndCollapseMap(config map[string]interface{}, path []string) {
+	key := path[0]
```

Comment:
Can we check for the possibility of an empty path and empty map before this to avoid panics during runtime?

Reply:
Done ✅